packages/realm/binding/apple/NativeRealmLoader-cxx.mm

```diff
@@ -0,0 +1,40 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2016 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+#import "NativeRealmLoader.h"
+#import "NativeRealmModule.h"
+
+#import <jsi_init.h>
+
+#import <ReactCommon/CallInvoker.h>
+#import <ReactCommon/CxxTurboModuleUtils.h>
+
+namespace react = facebook::react;
+
+@interface NativeRealmLoader()
+@end
+
+@implementation NativeRealmLoader
```

Comment:
Wondering if `RCT_EXPORT_MODULE()` is also necessary here based on [this example](https://github.com/reactwg/react-native-new-architecture/blob/main/docs/turbo-modules.md#rtncalculatormm).